templates/partials/answers/address.html

```diff
@@ -0,0 +1,35 @@
+{% from "components/address/_macro.njk" import onsAddressInput %}
+
+{% set input = form.fields[answer.id] if input is undefined else input %}
+
+{% set config = {
+    "id": "address",
+    "legend": question.title,
+    "legendClasses": "u-vh",
+    "autosuggest":{
+        "label": {
+            "text": _("Enter address or postcode and select from results")
+        },
+    },
```

Comment:
This was added because the DS component requires this at the moment. This bug will be fixed in the DS soon.